app/models/container_label_tag_mapping/mapper.rb

```diff
@@ -16,7 +16,8 @@ def initialize(mappings)
       @mappings = mappings.group_by { |m| [m.label_name, m.labeled_resource_type, m.label_value].freeze }
                           .transform_values { |ms| ms.collect(&:tag_id) }
 
-      @tags_to_resolve_collection = ::ManagerRefresh::InventoryCollection.new(
+      require "inventory_refresh"
```

Comment:
Is there a potential timing issue with this?  The provider repos are not requiring inventory_refresh gem (though maybe they should?), and if they reference Inventory::Refresh before it's required, they will get an error.

Reply:
Less so here, but the question is more in general.

Reply:
True, every place the provider references InventoryRefresh is from a refresh context so I figured [this require](https://github.com/ManageIQ/manageiq/pull/17925/files#diff-0faa6f426ffeb5bf0cf6ee8c2ecae2e2R80) at the start of refresh would cover those cases.